### PR TITLE
[release/v2.22] Bump version to v2.22.13

### PR DESCRIPTION
**Pull request by akash-gautam**
**What this PR does / why we need it**:
Bumps the version to v2.22.13
**What type of PR is this?**
/kind chore

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change? Then add your Release Note here**:

```release-note
NONE
```

**Documentation**:

```documentation
NONE
```

### CODE REVIEW

File: modules/web/src/assets/config/changelog.json [hunk 1]
```diff
@@ -1,17 +1,4 @@
 {
   "externalChangelogURL": "https://github.com/kubermatic/kubermatic/blob/main/docs/changelogs/CHANGELOG-2.22.md#v22212",
```
Comment (archups) [inline modules/web/src/assets/config/changelog.json:2]: update `v22212` to `v22213`

**Comment by archups** [comment]
@Waseem826

File: modules/web/src/assets/config/changelog.json [hunk 2]
```diff
@@ -1,17 +1,4 @@
...
-      "category": "changed",
-      "description": "Update EKS/AKS version matrix to only include Kubernetes versions supported by those managed offerings. For  AKS 1.26-1.28 are supported, for EKS 1.24 to 1.28. The default for newly created external clusters is now 1.28."
-    },
-    {
-      "category": "fixed",
-      "description": "Fix a bug where Operating System Profiles were not being listed for GCP/GCE."
-    }
-  ]
+  "externalChangelogURL": "https://github.com/kubermatic/kubermatic/blob/main/docs/changelogs/CHANGELOG-2.22.md#v22213",
+  "entries": []
```
Comment (Waseem826) [inline modules/web/src/assets/config/changelog.json:3]: I think there should be something in the changelog entries. Or all changes only related to backend?

Comment (akash-gautam) [reply modules/web/src/assets/config/changelog.json:3]: For this patch release, there are no changes that will be visible on the UI so we keep it empty.

**Review by ahmedwaleedmalik — APPROVED** [review]
/approve

**Pull request merged**